poller/poller.go

```diff
@@ -0,0 +1,65 @@
+package poller
+
+import (
+	"time"
+
+	"github.com/godaddy/split-go-serializer/api"
+)
+
+// Poller contains cache data and SplitioAPIBinding
+type Poller struct {
+	Cache              int
```

Comment:
make it temporarily int, eventually it should probably be some type of (Splits + Segments) defined in `api`